src/main/java/frc/robot/canrange/CANrangeIO.java

```diff
@@ -0,0 +1,13 @@
+package frc.robot.canrange;
+
+import org.littletonrobotics.junction.AutoLog;
+
+public interface CANrangeIO {
+
+  @AutoLog
+  public static class CANrangeIOInputs {
+    public double distanceM = 0.0;
```

Comment:
```suggestion
    public double distanceMeters = 0.0;
```
i _think_ it's in meters??